readthedocs/projects/tasks/builds.py

```diff
@@ -940,7 +945,9 @@ def send_notifications(self, version_pk, build_pk, event):
     bind=True,
     ignore_result=True,
 )
-def update_docs_task(self, version_id, build_id, build_commit=None, **kwargs):
+def update_docs_task(
+    self, version_id, build_id, *, build_api_key, build_commit=None, **kwargs
```

Comment:
I'm also +1 on passing everything as kwargs, just avoiding having to change the tests here. But also, not sure if this will be a breaking change since we are reading from args here

https://github.com/readthedocs/readthedocs.org/blob/f261ecc69884421332c93f007532ef63ada35f36/readthedocs/projects/tasks/builds.py#L386-L386